tests/Library/Vanilla/Embeds/EmbedManagerTest.php

```diff
@@ -100,6 +102,27 @@ public function provideRenderedData() {
                 ],
 '<div class="embed embedInstagram">
     <blockquote class="instagram-media" data-instgrm-captioned data-instgrm-permalink="https://www.instagram.com/p/BizC-PPFK1m" data-instgrm-version="8"/>
+</div>'
+            ],            [
+                [
+                    "url" =>"https://soundcloud.com/syrebralvibes/the-eden-project-circles",
```

Comment:
I'm cool with using double quotes or single quotes, but let's stick with one or the other in here. Some lines are using double quotes. Some are using single quotes. Some are using a mix, with no apparent reason why.